app/views/hub/ctc_clients/new.html.erb

```diff
@@ -8,20 +8,22 @@
           <h1 class="form-card__title">
             <%= @title %>
           </h1>
-          <%= f.cfa_select(:service_type, t("hub.clients.fields.service_type"), [["Drop off", "drop_off"]], options: { selected: "drop_off" } ) %>
+          <%= f.hidden_field :service_type, value: "drop_off" %>
```

Comment:
Move service_type to a hidden field since the value is always constant -- no reason for a dropdown.